java/custom/com/sun/cldc/i18n/j2me/UTF_8_Reader.java

```diff
@@ -32,20 +32,62 @@
 
 /** Reader for UTF-8 encoded input streams. */
 public class UTF_8_Reader extends StreamReader {
```

Comment:
If you have restored this file to the original version, I think we can just remove it from java/custom/